lib/utils.sh

```diff
@@ -64,38 +70,41 @@ check_if_version_exists() {
   local plugin_name=$1
   local version=$2
 
-  check_if_plugin_exists $plugin_name
+  check_if_plugin_exists "$plugin_name"
 
-  local install_path=$(find_install_path $plugin_name $version)
+  local install_path
+  install_path=$(find_install_path "$plugin_name" "$version")
 
-  if [ "$version" != "system" ] && [ ! -d $install_path ]; then
+  if [ "$version" != "system" ] && [ ! -d "$install_path" ]; then
     display_error "version $version is not installed for $plugin_name"
     exit 1
   fi
 }
 
 get_plugin_path() {
-  echo $(asdf_dir)/plugins/$1
+  echo "$(asdf_dir)"/plugins/"$1"
```

Comment:
Again, it's easier if you wrap the whole path in a single set of double quotes.

Reply:
ditto.